index.html

```diff
@@ -1 +1,36 @@
-Hello World
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Lil Boat</title>
+</head>
+<body>
+<h1>Lil Yachty</h1>
```

Comment:
Layout looks good.  Make sure you are indenting your code to fit the nesting.  All of the content below body should be nested within body and indented properly